app/src/main/admin/components/ModellingGroups/Models/ModelMetaTable.tsx

```diff
@@ -102,20 +102,20 @@ export class ModelMetaTableComponent extends React.Component<ModelMetaProps, Sta
             <table>
                 <thead>
                 <tr>
-                    {this.createHeader("modelling_group", "Group")}
-                    {this.createHeader("id", "Model Name")}
-                    {this.createHeader("disease", "Disease")}
-                    {this.createHeader("is_dynamic", "Model Type")}
-                    {this.createHeader("scenario_count", "Scenarios")}
-                    {this.createHeader("code", "Code")}
-                    {this.createHeader("gender", "Gender")}
-
-                    {this.createHeader("max_countries", "Max Countries")}
-                    {this.createHeader("years", "Years")}
-                    {this.createHeader("ages", "Ages")}
-                    {this.createHeader("cohorts", "Cohorts")}
-                    {this.createHeader("outcomes", "Outcomes", true)}
-                    {this.createHeader("has_dalys", "DALYs")}
+                    {this.createHeader("modelling_group", "Group", "7em")}
```

Comment:
Maybe there's a cleverer way to do this but I couldn't think of one! This forces the table to scroll at narrow widths